src/app/profile-update/profile-update.page.ts

```diff
@@ -58,12 +58,32 @@ export class ProfileUpdatePage implements OnInit {
 
 
   getProfileData() {
+    let state;
     this.toast.startLoader();
     const config = {
       url: `${urlConstants.API_URLS.GET_PROFILE}`
     }
     this.showLoader = true;
     this.kendraService.get(config).subscribe(data => {
+      if (data.result.roles && data.result.roles.length) {
```

Comment:
Is this completely working? I f not please remove this change

Reply:
working, i checked that issue is resolved from backend as well